src/streaming/text/NotFragmentedTextBufferController.js

```diff
@@ -28,19 +28,20 @@
  *  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  *  POSSIBILITY OF SUCH DAMAGE.
  */
-import Constants from '../constants/Constants';
 import EventBus from '../../core/EventBus';
 import Events from '../../core/events/Events';
 import FactoryMaker from '../../core/FactoryMaker';
 import InitCache from '../utils/InitCache';
+import SourceBufferSink from '../SourceBufferSink';
+import TextController from '../../Streaming/Text/TextController';
```

Comment:
Paths are case sensitive on some platforms. This seems to be causing the tests to fails.